tests/test_views.py

```diff
@@ -19,9 +19,7 @@ def test_main_page_(self):
     def test_main_page_paginated_correctly(self):
         response = self.client.get(MAIN_PAGE_URL)
 
-        self.assertEqual(
-            len(response.context["post_list"]), PAGINATION
-        )
+        self.assertEqual(len(response.context["post_list"]), PAGINATION)
```

Comment:
Ensure that the context variable is named `object_list` or `page_obj.object_list` as per Django's ListView default context naming, unless explicitly overridden in the view. Adjust the test accordingly if needed.